tests/TestCoreFn.hs

```diff
@@ -47,6 +47,15 @@ spec = context "CoreFnFromJsonTest" $ do
       ss = SourceSpan mp (SourcePos 0 0) (SourcePos 0 0)
       ann = ssAnn ss
 
+  specify "should parse version" $ do
+    let v = Version [0, 13, 6] []
```

Comment:
With the old implementation, using `let v = Version [0] []` here still allowed this test to pass.